.github/workflows/render-website.yaml

```
@@ -27,8 +27,19 @@ jobs:
           git config --global user.name "$GITHUB_ACTOR"
           git config --global user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
+      - name: Install git-extras
+        run: |
+          git clone https://github.com/tj/git-extras.git
+          cd git-extras
+          git checkout $(git describe --tags $(git rev-list --tags --max-count=1))
+          sudo make install
```

Slightly better on your own machine, doesn't make a difference here:

```suggestion
          make && sudo make install
```

---

Ouch.

I mean, normally we'd `make` and `sudo make install` ...